src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -101,6 +112,12 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         /// <remarks>This property is used only for unit testing.</remarks>
         internal IConfigurationClientManager ClientManager { get; set; }
 
+        /// <summary>
+        /// An optional class used to process pageable results from Azure App Configuration.
+        /// </summary>
+        /// <remarks>This property is only set outside of this class if it's used for unit testing.</remarks>
```

Comment:
I would merge the remarks with the summary since this is internal. Additionally, I would put For testing purposes as the first part of the sentence.